### DISCUSSION TITLE
Can I locate GitHub Actions workflow files in a place other than .github/workflows?

### DISCUSSION THREAD

**Discussion by dongho-jung** [community/community#28956] [Other Feature Feedback, Questions, & Ideas] [answered]
Can I locate GitHub Actions workflow files in a place other than .github/workflows?

I'm migrating a monorepo relying on CodeBuild to GitHub Actions, and the monorepo has several components and the each component has its own build spec(it's something like GitHub Actions workflow file) I want to make changes as least as possible so I want to know GitHub Actions can support this(locate workflow files elsewhere)

**Comment by airtower-luna** [comment, marked as answer]
No, that's not supported. Custom *actions* can be anywhere, but workflows must be in `.github/workflows/`.
